app/controllers/treks_controller.rb

```diff
@@ -7,6 +7,8 @@ def show
     @trek = Trek.find(params[:id])
     @challenges = @trek.challenges
     @my_reports = Report.where(user: current_user, trek: @trek)
-    @reports = Report.where(trek: @trek, public: true)
+    @reports = Report
+      .where(trek: @trek, public: true)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align `.where` with `Report` on line 10.